lams_tool_scribe/web/pages/learning/scribe.jsp

```diff
@@ -123,7 +123,8 @@
 			}
 		
 			function confirmForceComplete() {
-				return confirm("<fmt:message key='message.confirmForceComplete'/>");
+				<%--return confirm("<fmt:message key='message.confirmForceComplete'/>");--%>
+				return confirm('fmt:message key="message.confirmForceComplete"/>');
```

Comment:
I believe your code misses an opening "<". It should be
```
return confirm('<fmt:message key="message.confirmForceComplete"/>');
```
Can you explain in English why this change is required? Does it cause a problem when it gets translated to other languages?